gnosis/safe-react
# Migrate owner management to sdk

## What it solves
Resolves #2796

## How this PR fixes it
Replaces the calls to add, remove and swap owners in a safe with the methods provided by the Core SDK

## How to test it
Add , remove and swap owner functionalities in the safe should work as expected.

## Screenshots
![Screen Shot 2021-10-14 at 15 23 29](https://user-images.githubusercontent.com/32431609/137326221-52759962-17d3-4dc7-9c0e-163098bf8bfa.png)
